pinot-hadoop/src/main/java/com/linkedin/pinot/hadoop/job/mapper/HadoopSegmentCreationMapReduceJob.java

```diff
@@ -170,6 +170,14 @@ private String createSegment(String dataFilePath, Schema schema, String seqId) t
 
       segmentGeneratorConfig.setOutDir(_localDiskSegmentDirectory);
 
+      // Add the current java package name and version to the segment metadata
+      // properties file.
+      Package objPackage = this.getClass().getPackage();
+      String packageName = objPackage.getSpecificationTitle();
+      String packageVersion = objPackage.getSpecificationVersion();
+      LOGGER.info("Pinot Hadoop Package Name {} and version {}", packageName, packageVersion);
+      segmentGeneratorConfig.setCreatorVersion(packageName + packageVersion);
```

Comment:
This will look like `pinot-hadoop0.1.2` which looks kind of weird. I'd add a hyphen.